### DISCUSSION TITLE
Add support for Trading 212 import

### DISCUSSION THREAD

**Discussion by dickwolff** [ghostfolio/ghostfolio#1470] [Ideas]
Hi,

I wish to import my trades from Trading 212. You can export transactions from the app, which will result in a CSV file in the format like below. 

```
Action,Time,ISIN,Ticker,Name,No. of shares,Price / share,Currency (Price / share),Exchange rate,Total (EUR),Withholding tax,Currency (Withholding tax),ID
Market buy,2022-03-28 10:27:15,NL0010273215,ASML,"ASML Holding",0.0021880000,621.50,EUR,1.00000,1.36,,,EOF1848582542
Dividend (Ordinary),2022-05-16 09:37:52,NL0010273215,ASML,"ASML Holding",0.0021880000,3.15,EUR,Not available,0.01,0.00,EUR,
```

You can extract the action, ticker, amount and price from the file.

Would it be possible to add support for importing Trading 212 transactions? Currently I would have to convert all these transactions to the Ghostfolio format, which is quite a chore.

Thanks!

**Comment by dickwolff** [comment]
I have created a simple tool to help me bypass this problem for now. I put it on my [GitHub](https://github.com/dickwolff/Trading-212-to-Ghostfolio), so feel free to check it out if this is something someone else wants to use.

@dtslvr Feel free to use this as an example as to how this could be implemented in Ghostfolio, should you ever choose to do so! It isn't the prettiest code I have ever written but it works for now 😉

**Reply by dtslvr** [reply]
Creating a separate converter script for bank / broker X is currently the best solution. The focus of Ghostfolio remains on the core functionality.

Thanks for sharing @dickwolff.
